backend/app/main.py

```
@@ -28,22 +28,40 @@
 import uvicorn
 import dotenv
 import sentry_sdk
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)